### PR TITLE
Updating branch by coments fixed

### PR DESCRIPTION
**Pull request by Linases**
image

### CODE REVIEW

File: Constants/Admin/AdminTabNames.cs [hunk 1]
```diff
@@ -1,13 +1,13 @@
-﻿namespace Constants
+﻿namespace Constants.Admin
 {
-    public class AdminElementsNames
+    public class AdminTabNames
     {
         public const string UserManagement = "User Management";
         public const string Job = "Job";
         public const string Organization = "Organization";
-        public const string Qalifications = "Qalifications";
```
Comment (ashurakova) [inline Constants/Admin/AdminTabNames.cs]: remove unused variables

File: Constants/Performance/ConfigureMenuNames.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Constants.Performance;
+
+public class ConfigureMenuNames
+{
+    public const string KPIs = "KPIs";
+    public const string Trackers = "Trackers";
```
Comment (ashurakova) [inline Constants/Performance/ConfigureMenuNames.cs]: Trackers not used

File: Constants/PIM/PIMHeadersNames.cs [hunk 3]
```diff
@@ -1,8 +1,8 @@
-﻿namespace Constants
+﻿namespace Constants.PIM
 {
-    public class PIMHeadersNames
+    public class PimTabNames
     {
         public const string AddEmployee = "Add Employee";
         public const string EmployeeList = "Employee list";
```
Comment (ashurakova) [inline Constants/PIM/PIMHeadersNames.cs]: variable not used

File: FactoryPattern/BrowserFactory.cs [hunk 4]
```diff
@@ -1,7 +1,9 @@
-﻿using Constants;
+using Constants;
+using Constants.TestSettings.Enum;
```
Comment (ashurakova) [inline FactoryPattern/BrowserFactory.cs]: unused namespaces

File: Orange_HRM_Pages/BasePage.cs [hunk 5]
```diff
@@ -1,42 +1,56 @@
...
-        protected static By SaveButtonLocator => By.XPath("//button[text()=' Save ']");
-        protected static By AddButtonLocator => By.XPath("//button[text()=' Add ']");
-        protected static By VerifyDelete => By.XPath("//*[@class='oxd-icon bi-trash oxd-button-icon']");
-        protected static Button AddButton => new Button(AddButtonLocator);
-        protected static Button SaveButton => new Button(SaveButtonLocator);
-        protected static Button VerifyDeleteButton => new Button(VerifyDelete);
-        protected HrmWebElement Spinner => new HrmWebElement(SpinnerLocator);
-
-        public void ClickAddButton()
+        protected const string TableListLocator = "//*[@class='oxd-table-card']";
```
Comment (ashurakova) [inline Orange_HRM_Pages/BasePage.cs:12]: variables should go in the following order:
private 
protected
public

File: TestSettingsConfiguration/TestSettings.cs [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    public class TestSettings
+    {
+        private const string TestSettingsJson = "testsettings.json";
+
+        public TestSettings()
+        {
+            SetDefaultValues();
+        }
+
+        private IConfiguration TestConfiguration { get; } =
```
Comment (ashurakova) [inline TestSettingsConfiguration/TestSettings.cs]: use can use one line here

File: Wrappers/Button.cs [hunk 7]
```diff
@@ -6,29 +6,12 @@ namespace Wrappers
 {
     public class Button : HrmWebElement
     {
-        public Button(By locator) : base(locator)
-        {
-        }
+        private Button ButtonElement => new(By);
```
Comment (ashurakova) [inline Wrappers/Button.cs]: no references

File: Constants/Admin/AdminTabNames.cs (hunk 1 above)
Comment (Linases) [reply Constants/Admin/AdminTabNames.cs]: Removed

File: Constants/Performance/ConfigureMenuNames.cs (hunk 2 above)
Comment (Linases) [reply Constants/Performance/ConfigureMenuNames.cs]: removed

File: Constants/PIM/PIMHeadersNames.cs (hunk 3 above)
Comment (Linases) [reply Constants/PIM/PIMHeadersNames.cs]: Removed

File: FactoryPattern/BrowserFactory.cs (hunk 4 above)
Comment (Linases) [reply FactoryPattern/BrowserFactory.cs]: removed unused

File: Orange_HRM_Pages/BasePage.cs (hunk 5 above)
Comment (Linases) [reply Orange_HRM_Pages/BasePage.cs:12]: private variable moved first

File: TestSettingsConfiguration/TestSettings.cs (hunk 6 above)
Comment (Linases) [reply TestSettingsConfiguration/TestSettings.cs]: made it in one line

File: Wrappers/Button.cs (hunk 7 above)
Comment (Linases) [reply Wrappers/Button.cs]: removed

**Comment by ashurakova** [comment]
approved for merge

**Pull request merged**